README.md

```diff
@@ -22,13 +22,9 @@ you fix issues as fast as possible.
 - Remediation advice
 - License compliance
 
-## Supported package managers
-
-- NPM, Yarn, Yarn Workspaces
-- Maven, Gradle
-- NuGet, Paket
-- Pip, PyPI, Pipenv
-- Cocoapods
-- Composer
+## Useful links
+
+- We support languages like Java, JavaScript, .NET. See more at [Snyk page](https://snyk.co/ucWSd)
```

Comment:
Looking at it again, I'd update it a bit:

```suggestion
- This plugin works with projects written in Java, JavaScript, .NET and many more languages. See the [full list of languages and package managers Snyk supports](https://snyk.co/ucWSd)
```

It makes it more obvious "there is more" and it makes the actual link text more descriptive

Reply:
I agree, now it sounds better!